frontend/src/_routes.js

```diff
@@ -1,17 +1,20 @@
 import Login from "./views/Core/Login/Login";
 import Dashboard from "./views/Core/Dashboard/Dashboard";
+import ForgotPassword from "./views/Core/ForgotPassword/ForgotPassword";
 import GithubAuthProviderHook from "./core/components/providers/auth/GithubAuthProviderHook";
 import GoogleAuthProviderHook from "./core/components/providers/auth/GoogleAuthProviderHook";
 
 export const LOGIN_PATH = "/login";
 export const HOME_PATH = "/dashboard";
+export const FORGOTPASSWORD_PATH = "/forgot-password";
```

Comment:
For these ones, please refactoring to a JSON variable named routePaths, with the following structure:

```javascript
export const routePaths = {
login: "/login",
home: "/home",
forgot_password: "/forgot-password"
.
.
.
};
```
Then change all old references for the new ones.